src/content/developers/docs/apis/javascript/index.md

```diff
@@ -11,6 +11,8 @@ For this purpose, every Ethereum client implements the [JSON-RPC](/developers/do
 
 If you want to use JavaScript to connect with an Ethereum node, it's possible to use vanilla JavaScript but several convenience libraries exist within the ecosystem that make this much easier. With these libraries, developers can write intuitive, one-line methods to initialize JSON RPC requests (under the hood) that interact with Ethereum.
 
+Please note that since [The Merge](/upgrades/merge) two connected pieces of Ethereum software - an execution client and a consensus client - are required to run a node. This page assumes that both clients are installed and running correctly. For more information please see [Run a Node](/developers/docs/nodes-and-clients/run-a-node).
```

Comment:
```suggestion
Please note that since [The Merge](/upgrades/merge/), two connected pieces of Ethereum software - an execution client and a consensus client - are required to run a node. This page assumes that both clients are installed and running correctly. For more information please see [Run a Node](/developers/docs/nodes-and-clients/run-a-node/).
```